pkg/accountmanager/manager.go

```diff
@@ -241,7 +235,7 @@ func (a *AccountManager) addAccountPoller(cloudInterface common.CloudInterface,
 func (a *AccountManager) removeAccountPoller(namespacedName *types.NamespacedName) error {
 	accPoller, exists := a.getAccountPoller(namespacedName)
 	if !exists {
-		return nil
+		return fmt.Errorf(fmt.Sprintf("%v %v", errorMsgAccountPollerNotFound, namespacedName))
```

Comment:
why this error message. if it doesnt exist, let it be, as its remove case

Reply:
On the CPA remove account, we will be ignoring any error returned as part of removeAccountPoller.

Going by the function name removeAccountPoller - expect to remove the poller.  If for some reason, we are unable then we can return the error to the caller so that he can chose to ignore/handle it.